app/controllers/users/users.controller.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../../models/index');
+const logger = require('../../logger');
+
+const { databaseError } = require('../../errors');
+const { encryptar } = require('./utils');
+
+exports.createUser = (req, res) => {
+  const { name, last_name, email, password } = req.body;
+  encryptar(password)
```

Comment:
I think that the controller is very overloaded with logic and at first glance it is difficult to understand what is being done, so it is advisable to separate it in this case to take it to a service